saleor/graphql/order/mutations/fulfillments.py

```diff
@@ -622,22 +646,138 @@ class Meta:
         error_type_class = OrderError
         error_type_field = "order_errors"
 
+    @classmethod
+    def _check_shipping_costs(cls, payments_to_refund):
+        num_of_occurances = 0
+        for payment in payments_to_refund:
+            if payment.get("include_shipping_costs"):
+                num_of_occurances += 1
+
+        if num_of_occurances > 1:
+            raise ValidationError(
+                {
+                    "include_shipping_costs": ValidationError(
+                        "Shipping costs cannot be included in more than one payment.",
+                        code=OrderErrorCode.DUPLICATED_INPUT_ITEM,
+                    )
+                }
+            )
+
+    @classmethod
+    def _check_payments_belong_to_order(cls, order, payments_ids) -> None:
+        order_payments_ids = order.payments.filter(id__in=payments_ids).values_list(
+            "id", flat=True
+        )
+        if set(order_payments_ids) != set(payments_ids):
+            improper_payments_ids = set(payments_ids).difference(
+                set(order_payments_ids)
+            )
+            improper_payments_global_ids = [
+                graphene.Node.to_global_id("Payment", payment_id)
+                for payment_id in improper_payments_ids
+            ]
+            code = OrderErrorCode.PAYMENTS_DO_NOT_BELONG_TO_ORDER
+            raise ValidationError(
+                {
+                    "payments_to_refund": ValidationError(
+                        "These payments do not belong to the order.",
+                        code=code,  # type: ignore
```

Comment:
Instead of adding `# type: ignore` just get `value` from code.
`code = OrderErrorCode.PAYMENTS_DO_NOT_BELONG_TO_ORDER.value`

Reply:
Thanks, added.
Any idea why it didn't complain about other similar lines? As in other cases, the value isn't passed.

Reply:
It seems like it checks types only in methods where the type hints are defined.

Reply:
And, I didn't see the change 🤔